apps/ejabberd/priv/mysql.sql

```diff
@@ -303,7 +303,8 @@ CREATE INDEX i_muc_light_blocking USING HASH ON muc_light_blocking(luser, lserve
 
 CREATE TABLE muc_room (
     name        VARCHAR(250) NOT NULL,
-    host        VARCHAR(250) NOT NULL,
+    -- According to RFC 1035 the length of a FQDN is limited to 255 characters
```

Comment:
Maybe we could use only a small prefix of this column for an index? Even when room names get duplicated on many hosts, it will still be cheap to scan couple of rows.

OR we could index a prefix of `name` and use full `host`. Or use prefixes in both cases.